src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -3,13 +3,17 @@
 //
```

Comment:
The file uses new lines between statements. Let's preserve the styling with our new additions.

Reply:
Sorry, I'm not sure I fully grasp your suggestion. Could you please clarify this?